gcp/gcp.go

```diff
@@ -109,9 +114,10 @@ func (b *Backend) Receive(ctx context.Context, numMessages uint32, visibilityTim
 					message.DeliveryAttempt = &deliveryAttemptDefault
 				}
 				metadata := Metadata{
-					pubsubMessage:   message,
-					PublishTime:     message.PublishTime,
-					DeliveryAttempt: *message.DeliveryAttempt,
+					pubsubMessage:    message,
+					PublishTime:      message.PublishTime,
+					DeliveryAttempt:  *message.DeliveryAttempt,
+					SubscriptionName: strings.Replace(subID, "hedwig", "", 1),
```

Comment:
Huh shouldn't it be "hedwig-". And maybe to be defensive it should be remove prefix not replace all.

Reply:
ah yeah I caught that in the test and just changed - but will change to only remove at the beginning